scripts/e2e-monolith.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -x
+
+# Monolith version to test against
+export BUILDVERSION=2022.09.0-begin-289-g7f5834bbae22
+
+export CI=true
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/junit"
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
+export ORG_OIE_ENABLED=true
+
+set +e
+source $OKTA_HOME/$REPO/scripts/setup.sh
+set -e
+cd $OKTA_HOME/$REPO
+
+# Start monolith
+create_log_group "Start Monolith"
+source ./scripts/monolith/install-dockolith.sh
+source ./scripts/monolith/start.sh
+finish_log_group $?
+
+# Create test org and save environment variables in "testenv"
+create_log_group "Create Test Org"
+# Add widget test host to /etc/hosts
+export TEST_ORG_SUBDOMAIN="siw-test-1"
+echo "${DOCKER_HOST_CONTAINER_IP} ${TEST_ORG_SUBDOMAIN}.okta1.com" >> /etc/hosts
+cat /etc/hosts
+source ./scripts/monolith/create-testenv.sh
+export ORG_OIE_ENABLED=true
+finish_log_group $?
+
+# Build Widget
+create_log_group "Build Widget"
+if ! yarn build:release; then
+  echo "build failed! Exiting..."
+  exit ${TEST_FAILURE}
+fi
+finish_log_group $?
+
+# Setup E2E environment
+create_log_group "E2E Setup"
+set +e
+setup_service google-chrome-stable 89.0.4389.72-1
```

Comment:
nit: I think we have later versions of chrome available. Let me find out which one is the latest

Reply:
[105.0.5195.125-1](https://artifacts.aue1e.internal/artifactory/thirdparty-yum/x86_64/chrome-sel3/105.0.5195.125-1/) is the latest available. We can try running against this and see if it works.

Reply:
Until this script replaces the default E2E script I'd like to minimize variables and keep it as close as possible to the default E2E. So if we upgrade chrome version, it should be across the board. I created a task for this: OKTA-535569